ZukoAzula2016/src/org/team1540/zukoazula/Portcullis.java

```diff
@@ -0,0 +1,48 @@
+package org.team1540.zukoazula;
+
+import ccre.behaviors.ArbitratedFloat;
+import ccre.channel.BooleanCell;
+import ccre.channel.BooleanInput;
+import ccre.channel.EventOutput;
+import ccre.channel.FloatCell;
+import ccre.channel.FloatIO;
+import ccre.channel.FloatInput;
+import ccre.cluck.Cluck;
+import ccre.ctrl.ExtendedMotorFailureException;
+import ccre.ctrl.PIDController;
+import ccre.drivers.ctre.talon.TalonExtendedMotor;
+import ccre.frc.FRC;
+
+public class Portcullis {
+
+    private static final TalonExtendedMotor leftGrabMotor = FRC.talonCAN(12);
+    private static final TalonExtendedMotor rightGrabMotor = FRC.talonCAN(13);
+
+    private static final FloatIO leftEncoder = leftGrabMotor.modEncoder().getEncoderPosition();
+    private static final FloatIO rightEncoder = rightGrabMotor.modEncoder().getEncoderPosition();
+
+    private static final FloatInput control = ZukoAzula.controlBinding.addFloat("Portcullis Grabber Axis").deadzone(0.2f);
```

Comment:
Shouldn't we put deadzones on our joystick controls just as we did with our drive code?